content/realtime/channels.textile

```diff
@@ -813,6 +814,55 @@ Previously registered listeners can be removed individually or all together.
     stateChangeListener.cancel();
 ```
 
+
+h3(#channel-detach). Detaching from a channel
+
+A client can detach from a channel so that they no longer receive any messages published to that channel. Detaching is different to unsubscribing from a channel as "@unsubscribe()@":#unsubscribe is a client-side operation, meaning Ably is unaware of whether or not a client has unsubscribed from the channel. Messages will continue to be streamed to the client until "@detach()@":#detach is called.
```

Comment:
Suggest referring to a client as "it" to make it clear that we're talking about a bit of code rather than a company that pays for our services?

```suggestion
A client can detach from a channel so that it no longer receives any messages published to that channel. Detaching is different to unsubscribing from a channel as "@unsubscribe()@":#unsubscribe is a client-side operation, meaning Ably is unaware of whether or not a client has unsubscribed from the channel. Messages will continue to be streamed to the client until "@detach()@":#detach is called.
```

Reply:
Perhaps we can simplify this sentence, for example:

"A client can detach from a channel so that it no longer receives any messages published to that channel. Detaching is different to unsubscribing from a channel because "@unsubscribe()@":#unsubscribe is a client-side operation. The Ably platform does not know that a client has unsubscribed and will continue to stream messages to that client until "@detach()@":#detach is called."

Reply:
Good point on the "it" and "client" point. I've taken your suggestion for the rewrite as it's far less wordy!